DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/output_files/add.sta.summary

```diff
@@ -0,0 +1,55 @@
+------------------------------------------------------------
```

Comment:
Please remove this file, as well as add.sta.rpt 
and I hope that we can remove the add.sof file